### PR TITLE
Search bar component and key stats component added

### PR DESCRIPTION
**Pull request by slcfv3**
Card 382 finished and card 362 finished the key stats section.

### CODE REVIEW

File: stock-tracker/src/reducers/rootReducer.js [hunk 1]
```diff
@@ -29,12 +44,21 @@ export const rootReducer = (state = initialState, action) => {
...
         case 'PRICE_RECEIVED':
-            console.log("price in PRICE_RECEIVED:", action.payload.latestPrice)        
+            console.log("price in PRICE_RECEIVED:", action.payload.latestPrice)   
+             
             return {
                 ...state,
                 price: action.payload.latestPrice,
                 chart: action.payload.chart
             }
+        case 'STOCK_NOT_EXIST':
```
Comment (mhamdar) [inline stock-tracker/src/reducers/rootReducer.js:54]: I kind of disagree with these changes. Currently, if a user submits a stock that does not exist, he just gets an error indicating that, but it doesn't interrupt the current stock's data display / polling. I think this is the expected user behavior, i.e. you only move on to a next stock if a valid one is submitted.

With this change, however, you break the flow of data when an incorrect symbol is submitted, which I think is not ideal behaviour.

I'm open to having my mind changed on error handling, but in any case, a change in functionality like this should be a separate PR / branch to discuss.

File: stock-tracker/src/reducers/rootReducer.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 const initialState = {
-    symbol: "",
-    companyName: "",
+    symbol: "AAPL",
```
Comment (mhamdar) [inline stock-tracker/src/reducers/rootReducer.js:2]: Unfortunately, this introduces a bug: it won't fetch data if the first symbol a user submits is AAPL. I think the better approach might be to leave initial state as null, but maybe start up a fetch for AAPL data on startup (i.e. in our root saga).

File: stock-tracker/src/components/components.css [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+    position: relative;
+    left: 200px;
+}
+
+.pricetags{
+    position: relative;
+    left: 70%;
+    color: white;
+    font-size: 30px;
+}
```
Comment (mhamdar) [inline stock-tracker/src/components/components.css:71]: I don't have any issues with this.

File: stock-tracker/src/reducers/rootReducer.js (hunk 1 above)
Comment (slcfv3) [reply stock-tracker/src/reducers/rootReducer.js:54]: Ok, I am also trying to fix the bug. Now I got the reason, and I will modify it soon

**Pull request merged**